docs/usage/compact.rst.inc

```diff
@@ -66,11 +66,11 @@ Depending on the amount of segments that need compaction, it may take a while,
 so consider using the ``--progress`` option.
 
 A segment is compacted if the amount of saved space is above the percentage value
-given by the ``--threshold`` option. If ommitted, a threshold of 10% is used.
+given by the ``--threshold`` option. If omitted, a threshold of 10% is used.
```

Comment:
this is also auto-generated, see line 1 in this file. also check archiver.py.

Reply:
same for all other `docs/usage/*.rst.inc`.

Reply:
> this is also auto-generated, see line 1 in this file. also check archiver.py.

Yep, already fixed.

Reply:
> same for all other `docs/usage/*.rst.inc`.

Yep, already fixed also the docs/usage/ files

Reply:
i reverted this change in the autogenerated file.